### ISSUE TITLE
Not rendering correctly

### ISSUE THREAD

**Issue by Astuceman** [esemwy/Iray-Iridescent-Silk#1]
Hi

If think there is a problem with the shader

![mask](https://user-images.githubusercontent.com/19632209/40277343-1c6cd602-5c26-11e8-8fc7-10fc0f3dee45.png)

**Comment by esemwy** [comment]
Is there any useful information in the log file?

**Comment by Astuceman** [comment]
Yes
`2018-05-20 17:35:46.791 WARNING: dzneuraymgr.cpp(307): Iray ERROR - module:category(MDLC:COMPILER):   1.0   MDLC   comp error: could not find module '::local::iridescent_silk' in module path
2018-05-20 17:35:46.794 WARNING: dzneuraymgr.cpp(1189): MDL Import Error (4003) : The module "::local::iridescent_silk" was not found or failed to compile.
2018-05-20 17:35:46.794 WARNING: dzneuraymgr.cpp(1189): MDL Import Error (4006) : could not find module '::local::iridescent_silk' in module path
2018-05-20 17:35:46.798 WARNING: dzneuraymgr.cpp(307): Iray ERROR - module:category(MDLC:COMPILER):   1.0   MDLC   comp error: could not find module '::local::iridescent_silk' in module path
2018-05-20 17:35:46.798 WARNING: dzneuraymgr.cpp(1189): MDL Import Error (4003) : The module "::local::iridescent_silk" was not found or failed to compile.
2018-05-20 17:35:46.800 WARNING: dzneuraymgr.cpp(1189): MDL Import Error (4006) : could not find module '::local::iridescent_silk' in module path
2018-05-20 17:35:46.801 File loaded in 0 min 0.0 sec.
2018-05-20 17:35:46.808 Loaded file: Iridescent Silk.duf
`

**Comment by Astuceman** [comment]
Oups, sorry
So ... here the log 

`2018-05-20 17:35:46.791 WARNING: dzneuraymgr.cpp(307): Iray ERROR - module:category(MDLC:COMPILER):   1.0   MDLC   comp error: could not find module '::local::iridescent_silk' in module path`
`2018-05-20 17:35:46.794 WARNING: dzneuraymgr.cpp(1189): MDL Import Error (4003) : The module "::local::iridescent_silk" was not found or failed to compile.`
`2018-05-20 17:35:46.794 WARNING: dzneuraymgr.cpp(1189): MDL Import Error (4006) : could not find module '::local::iridescent_silk' in module path`
`2018-05-20 17:35:46.798 WARNING: dzneuraymgr.cpp(307): Iray ERROR - module:category(MDLC:COMPILER):   1.0   MDLC   comp error: could not find module '::local::iridescent_silk' in module path`
`2018-05-20 17:35:46.798 WARNING: dzneuraymgr.cpp(1189): MDL Import Error (4003) : The module "::local::iridescent_silk" was not found or failed to compile.`
`2018-05-20 17:35:46.800 WARNING: dzneuraymgr.cpp(1189): MDL Import Error (4006) : could not find module '::local::iridescent_silk' in module path`
`2018-05-20 17:35:46.801 File loaded in 0 min 0.0 sec.`
`2018-05-20 17:35:46.808 Loaded file: Iridescent Silk.duf`

I drag and drop the _Iridescent Silk.duf_ inside Daz

**Comment by esemwy** [comment]
You also have to set an “MDL path.” In Shader Mixer, Edit-> MDL Directory Manager. Alternatively you can copy the MDL file somewhere that’s already in your path. All three files are needed for the shader to function.

**Comment by Astuceman** [comment]
<img width="960" alt="capture 40" src="https://user-images.githubusercontent.com/19632209/40407411-033f9662-5e6d-11e8-85f9-50ef43584d7b.PNG">

Maybe it's not working because it's a 3delight shader ?
Your shader should looks like the first image I have posted ?

**Comment by esemwy** [comment]
No, it’s Iray MDL, actually just an example from the Nvidia documentation.

**Comment by esemwy** [comment]
The problem seems to be that more recent versions of DS disconnected the parameters from my custom MDL brick. If nothing is connected, you get white. I haven’t yet figured out how to re-expose the parameters.

**Comment by Astuceman** [comment]
Ok
Let me know if something new
